lib/services/ngsi/subscriptionService.js

```diff
@@ -96,24 +99,30 @@ function subscribe(device, triggers, content, callback) {
             'fiware-servicepath': device.subservice
         },
         json: {
-            entities: [
-                {
-                    type: device.type,
-                    isPattern: 'false',
-                    id: device.name
-                }
-            ],
-            reference: config.getConfig().providerUrl + '/notify',
-            duration: config.getConfig().deviceRegistrationDuration || 'P1M',
-            notifyConditions: [
-                {
-                    type: 'ONCHANGE',
-                    condValues: triggers
-                }
-            ],
-            throttling: config.getConfig().throttling
-        }
-    };
+                entities: [
+                    {
+                        type: device.type,
+                        isPattern: 'false',
+                        id: device.name
+                    }
+                ],
+                reference: config.getConfig().providerUrl + '/notify',
+                duration: config.getConfig().deviceRegistrationDuration || 'P1M',
```

Comment:
Yes, it is, but maybe not the appropriate one, since this is an automatic subscription. I'll change it to P100Y